gigascience/gigadb-website
# allowed to save species info with only minimum details

# Pull request for issue: #1871

removes a required attribute from an input that should not have it

## How to test?

* Go to http://gigadb.gigasciencejournal.com/adminSpecies/update/id/8, remove the content of the Genbank Name input and click save, it should submit the form
